test/integration/nss-tests.sh

```diff
@@ -3,12 +3,10 @@
 
 set -xo pipefail
 
-CA_PEM="$TEST_FIXTURES/ca.pem"
-CA_KEY="$TEST_FIXTURES/ca.key"
-
-CLIENT_CNF="$TEST_FIXTURES/client.cnf"
-PASSWORD_CA=whatever
 EXT_FILE="$TEST_FIXTURES/smimeextensions"
+CLIENT_CNF="$TEST_FIXTURES/client.cnf"
```

Comment:
`EXT_FILE` and `CLIENT_CNF` are overwritten in the call to `setup_ca` below. `CLIENT_CNF` remains the same, but `EXT_FILE` points to `xpextensions` instead. That's why `openssl ca` fails later on. I'd suggest to just remove both declarations here, and replace the deference of `$EXT_FILE` in the `openssl ca` call with `$TEST_FIXTURES/smimeextensions`.

Reply:
Thank you! I missed that